Cargo.toml

```diff
@@ -126,8 +126,8 @@ futures-bounded = { version = "0.2.4" }
 futures-rustls = { version = "0.26.0", default-features = false }
 getrandom = "0.2"
 if-watch = "3.2.1"
-hickory-proto = { version = "0.25.0-alpha.4", default-features = false }
-hickory-resolver = { version = "0.25.0-alpha.4", default-features = false }
+hickory-proto = { version = "0.25", default-features = false }
+hickory-resolver = { version = "0.25", default-features = false }
```

Comment:
I think it would be better to also specify the patch version

Reply:
Fixed.